### PR TITLE
feat(ST-2976): implement active column on company members page

### PR DESCRIPTION
**Pull request by Andrew-Orlov**
Description:

--

QA-test: ST-2979

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.6.0-pr-253-9cd089fe.zip

### CODE REVIEW

**Review by goldenmaya — CHANGES_REQUESTED** [review]
Screenshot 2022-08-02 at 13 37 15
Screenshot 2022-08-02 at 13 37 42
Screenshot 2022-08-02 at 13 43 04

there's no border and font-size 12px, also check text color, please

File: client-app/shared/account/composables/useOrganizationContacts.ts [hunk 1]
```diff
@@ -102,3 +102,26 @@ function getRoleName(contact: ContactType): string | undefined {
   }
   return roleName;
 }
+
+function getDisplayContactStatus(xapiStatus?: string): OrganizationContactDisplayStatusType {
```
Comment (artmakarov) [inline client-app/shared/account/composables/useOrganizationContacts.ts:106]: Move to `utils` folder

File: client-app/ui-kit/components/atoms/tooltip/vc-tooltip.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    aria-describedby="popover"
+    ref="triggerNode"
+    @mouseenter="toggleTooltip(true)"
+    @mouseleave="toggleTooltip(false)"
+  >
+    <slot name="trigger" />
+  </div>
+
+  <div class="bg-white border shadow-lg py-1.5 px-3.5" id="popover" ref="tooltipNode" v-show="isShown">
+    {{ content }}
```
Comment (artmakarov) [inline client-app/ui-kit/components/atoms/tooltip/vc-tooltip.vue:12]: Use `<slot />`

File: client-app/ui-kit/components/atoms/tooltip/vc-tooltip.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  tooltipInstance = createPopper(triggerNode.value!, tooltipNode.value!, {
+    placement: "bottom",
+    modifiers: [
+      {
+        name: "offset",
+        options: {
+          offset: [-15, 6],
+        },
+      },
+    ],
```
Comment (artmakarov) [inline client-app/ui-kit/components/atoms/tooltip/vc-tooltip.vue:43]: `vc-tooltip` should be customizable (props)

**Review by goldenmaya — APPROVED** [review]

**Review by artmakarov — APPROVED** [review]

**Pull request merged**